### PR TITLE
chore: fix ingress, improve dependencies check and add file header

### PR DESCRIPTION
**Pull request by evegufy**
## Description

- helm chart: change ingress to work without tls enabled
- dependencies-check: remove dash tool jar from repo: move to downloading the tool at every run
- add file headers #142

## Why

- change ingress because I noticed that it isn't possible to enable it without TLS enabled as well: this is quite an annoyance if one wants to use the chart in a local testing setup due to [this validation](https://github.com/eclipse-tractusx/policy-hub/blob/v1.0.0-rc.2/charts/policy-hub/templates/ingress.yaml#L50) being place which is taking effect if the default values aren't commented out.
-  change dependencies check due to: https://github.com/eclipse-tractusx/policy-hub/issues/127#issuecomment-2119967830

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/policy-hub/blob/main/docs/technical-documentation/dev-process/How%20to%20contribute.md)
- [x] I have performed a self-review of my own changes

### CODE REVIEW

File: charts/policy-hub/README.md [hunk 1]
```diff
@@ -6,12 +6,6 @@ For further information please refer to [Technical Documentation](../../docs/tec
 
 The referenced container images are for demonstration purposes only.
 
-## Prerequisites
```
Comment (tom-rm-meyer-ISST) [inline charts/policy-hub/README.md:9]: Prerequisites have been removed by regeneration / main has not yet been merged.

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Overall LGTM, please add prerequisites to your chart again. 

For helm check I just templated the chart with ingress.enabled = true and no error occurs (beside that the helm test workflow gonna test it).

Sidenote I missed previously, which I would not request as a point, but recommend: The system team provides an action for the changed dependency. You can refer to our workflow and increase the version of the dash tool [here](https://github.com/eclipse-tractusx/puris/blob/main/.github/workflows/dash-dependency-check.yml).

File: charts/policy-hub/README.md (hunk 1 above)
Comment (evegufy) [reply charts/policy-hub/README.md]: thank for point that out, I added the section now to `charts/policy-hub/README.md.gotmpl` so I don't overwrite it anymore with `helm-docs` anymore.

**Comment by evegufy** [comment]
> Overall LGTM, please add prerequisites to your chart again.
> 
> For helm check I just templated the chart with ingress.enabled = true and no error occurs (beside that the helm test workflow gonna test it).
> 
> Sidenote I missed previously, which I would not request as a point, but recommend: The system team provides an action for the changed dependency. You can refer to our workflow and increase the version of the dash tool [here](https://github.com/eclipse-tractusx/puris/blob/main/.github/workflows/dash-dependency-check.yml).

Good hint with the action from the systems team, I wasn't aware about that action, for now I'll leave the workflow as is because I have it running successfully in this manner a couple of other repos and I don't have time to rework it too much and don't want to spare such features like uploading the file if changed.

**Review by tom-rm-meyer-ISST — APPROVED** [review]
Smart! LGTM. The honor of merging remains at your side :)

**Pull request merged**